app/cdap/components/GridTable/index.tsx

```diff
@@ -57,6 +61,22 @@ export default function GridTable() {
     },
   ]);
 
+  const { dataprep } = DataPrepStore.getState();
+  const [isFirstWrangle, setIsFirstWrangle] = useState(false);
+  const [connectorType, setConnectorType] = useState<string | null>(null);
+  const [openDirectivePanel, setDirectivePanel] = useState(true);
```

Comment:
this should be something like directivePanelIsOpen and setDirectivePanelIsOpen - but either way the name should only differ by `set`

Reply:
Renamed